### PR TITLE
chore(core): removed site_id alias for site_guid in $CONFIG

### LINKED ISSUE #10468: remove $CONFIG->site_id as it is a duplicate of site_guid

### PR DESCRIPTION
**Pull request by jdalsem**
fixes #10468

### CODE REVIEW

File: mod/web_services/lib/web_services.php [hunk 1]
```diff
@@ -357,7 +357,7 @@ function api_auth_key() {
 	}
 
 	// check that it is active
-	$api_user = get_api_user($CONFIG->site_id, $api_key);
+	$api_user = get_api_user($CONFIG->site_guid, $api_key);
```
Comment (hypeJunction) [inline mod/web_services/lib/web_services.php]: Plugins shouldn't use `$CONFIG`.

File: docs/guides/upgrading.rst [hunk 2]
```diff
@@ -86,6 +86,7 @@ Removed global vars
 
  * ``$DEFAULT_FILE_STORE``
  * ``$SESSION``: Use the API provided by ``elgg_get_session()``
+ * ``$CONFIG->site_id``: Use ``$CONFIG->site_guid`` or ``elgg_get_site_entity()->guid``
```
Comment (hypeJunction) [inline docs/guides/upgrading.rst]: `elgg_get_config('site_guid')`

**Review by hypeJunction — CHANGES_REQUESTED** [review]

File: mod/web_services/lib/web_services.php (hunk 1 above)
Comment (jdalsem) [inline mod/web_services/lib/web_services.php]: just do a search for $CONFIG in the mod directory and you will have some extra work

Comment (jdalsem) [inline mod/web_services/lib/web_services.php]: not going to fix this as part of this PR...

**Comment by jdalsem** [comment]
addressed @hypeJunction 's comments

File: mod/web_services/lib/web_services.php (hunk 1 above)
Comment (jdalsem) [inline mod/web_services/lib/web_services.php]: Create https://github.com/Elgg/Elgg/issues/10470 to track this

**Review by mrclay — APPROVED** [review]

**Review by hypeJunction — APPROVED** [review]

**Pull request merged**